### ISSUE TITLE
[BUG] Rancher 2.6.6 is unable to provision a downstream RKE2 based cluster on minimal AlmaLinux 8.5 nodes

### ISSUE THREAD

**Issue by rajha-korithrien** [rancher/rancher#38676]
**Rancher Server Setup**
- Rancher version: 2.6.6
- Installation option: Helm Chart
   - Kubernetes Cluster and version: RKE2 v1.23.9+rke2r1
- Proxy/Cert Details: There is a tcp nginx proxy in front of the rancher/rke2 ingress nodes. Rancher uses a lets encrypt certificate (which works as expected).

**Information about the Cluster**
- Kubernetes version: v1.23.8+rke2r1
- Cluster Type (Local/Downstream): Downstream, bare metal and vms
* Custom - is provisioned by the provision command provided by the Rancher UI

**User Information**
- What is the role of the user logged in? Admin/Cluster

**Describe the bug**
We are unable to provision a downstream cluster when using the default configuration in the "Create Custom" cluster UI.
The first etcd/control node we add to the cluster never completes its provisioning.

**To Reproduce**
1) Install Rancher 2.6.6 onto an existing RKE2 cluster via helm
2) Use the Rancher UI to create a new downstream cluster (screen shot provides config)
3) Use the Rancher UI to get the provisioning command, set it to only create etcd, and control plane nodes
4) Run the provision command on a clean node that needs to have the etcd and control roles
5) Observe the node never provisions

**Result**
The UI provides the error message:
```
Error applying plan -- check rancher-system-agent.service logs on node for more information
```
For the first etcd/control node added.

**Expected Result**
The first node should bring up etcd such that we can join 2 more etcd/control nodes.
From there we should get a properly provisioned downstream cluster.

**Screenshots**
This shows the initial configuration used in the UI:
![image](https://user-images.githubusercontent.com/8145476/185653614-702d9807-f5fd-4f91-b413-c1b7eed7bbb3.png)

This shows the provision command used for the first node:
![image](https://user-images.githubusercontent.com/8145476/185653787-7cf2f14a-3b71-401d-813f-1a10bc86896a.png)

This shows the errors seen in the UI:
![image](https://user-images.githubusercontent.com/8145476/185653847-580ea563-94f6-49cf-abe4-322429e19062.png)

We have tried adding 2 more etcd/control nodes but they don't seem to do anything because the first etcd instance is not coming up.

**Additional context**
The logs from the `rancher-system-agent` on the first node:
```
-- Logs begin at Thu 2022-08-18 10:13:22 MDT, end at Fri 2022-08-19 08:48:11 MDT. --
Aug 19 08:47:42 control-0 systemd[1]: Started Rancher System Agent.
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Rancher System Agent version v0.2.7 (65aac83) is starting"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Using directory /var/lib/rancher/agent/work for work"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Starting remote watch of plans"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Starting /v1, Kind=Secret controller"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Detected first start, force-applying one-time instruction set"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="[Applyinator] Applying one-time instructions for plan with checksum 0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="[Applyinator] Extracting image rancher/system-agent-installer-rke2:v1.23.8-rke2r1 to directory /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d7>
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Using private registry config file at /etc/rancher/agent/registries.yaml"
Aug 19 08:47:42 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:42-06:00" level=info msg="Pulling image index.docker.io/rancher/system-agent-installer-rke2:v1.23.8-rke2r1"
Aug 19 08:47:44 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:44-06:00" level=info msg="Extracting file installer.sh to /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0/installer.sh"
Aug 19 08:47:44 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:44-06:00" level=info msg="Extracting file rke2.linux-amd64.tar.gz to /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0/rke2.linux-amd64.tar.gz"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="Extracting file sha256sum-amd64.txt to /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0/sha256sum-amd64.txt"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="Extracting file run.sh to /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0/run.sh"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[Applyinator] Running command: sh [-c run.sh]"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + SA_INSTALL_PREFIX=/usr/local"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + mkdir -p /var/lib/rancher/rke2"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + SAI_FILE_DIR=/var/lib/rancher/rke2/system-agent-installer"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + RESTART_STAMP_FILE=/var/lib/rancher/rke2/system-agent-installer/rke2_restart_stamp"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + RKE2_SA_ENV_FILE_NAME=rke2-sa.env"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + '[' '!' -d /var/lib/rancher/rke2/system-agent-installer ']'"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + mkdir -p /var/lib/rancher/rke2/system-agent-installer"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + check_target_mountpoint"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + mountpoint -q ''"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + check_target_ro"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + touch /usr/local/.rke2-ro-test"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + rm -rf /usr/local/.rke2-ro-test"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + test 0 -ne 0"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + SYSTEMD_BASE_PATH=/usr/local/lib/systemd/system"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + RKE2_SA_ENV_FILE_PATH=/var/lib/rancher/rke2/system-agent-installer/rke2-sa.env"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + RKE2_SA_ENV_SRV_REF=EnvironmentFile=-/var/lib/rancher/rke2/system-agent-installer/rke2-sa.env"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + '[' -f /var/lib/rancher/rke2/system-agent-installer/rke2_restart_stamp ']'"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + '[' -n 8173788393d2ab3786a09058c3b240c2a41f56f784f21553c5b76aac67326499 ']'"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + '[' '' '!=' 8173788393d2ab3786a09058c3b240c2a41f56f784f21553c5b76aac67326499 ']'"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + RESTART=true"
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: + env INSTALL_RKE2_ARTIFACT_PATH=/var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce6>
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stdout]: [INFO]  staging local checksums from /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a289097>
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stdout]: [INFO]  staging tarball from /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf>
Aug 19 08:47:49 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:49-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stdout]: [INFO]  verifying tarball"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stdout]: [INFO]  unpacking tarball file to /usr/local"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0:stderr]: /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6>
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[Applyinator] Command sh [-c run.sh] finished with err: <nil> and exit code: 127"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=error msg="error executing instruction 0: <nil>"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[Applyinator] No image provided, creating empty working directory /var/lib/rancher/agent/work/20220819-084742/0b0978d6824041704a2890977ce69ecf302376931d765bb07024859c87ff6c3b_0"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[Applyinator] Running command: sh [-c rke2 etcd-snapshot list --etcd-s3=false 2>/dev/null]"
Aug 19 08:47:50 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:50-06:00" level=info msg="[Applyinator] Command sh [-c rke2 etcd-snapshot list --etcd-s3=false 2>/dev/null] finished with err: <nil> and exit code: 127"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:47:51 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:51-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:47:56 control-0 rancher-system-agent[3417]: time="2022-08-19T08:47:56-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:48:01 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:01-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:48:06 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:06-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error encountered during parsing of last run time: parsing time \"\" as \"Mon Jan _2 15:04:05 MST 2006\": cannot parse \"\" as \"Mon\""
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error loading CA cert for probe (kube-scheduler) /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-scheduler.crt: open /var/lib/rancher/rke2/server/tls/kube-scheduler/kube-schedu>
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error while appending ca cert to pool for probe kube-scheduler"
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error loading x509 client cert/key for probe kube-apiserver (/var/lib/rancher/rke2/server/tls/client-kube-apiserver.crt//var/lib/rancher/rke2/server/tls/client-kube-apiserver.key): >
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error loading CA cert for probe (kube-controller-manager) /var/lib/rancher/rke2/server/tls/kube-controller-manager/kube-controller-manager.crt: open /var/lib/rancher/rke2/server/tls>
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error while appending ca cert to pool for probe kube-controller-manager"
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error loading CA cert for probe (kube-apiserver) /var/lib/rancher/rke2/server/tls/server-ca.crt: open /var/lib/rancher/rke2/server/tls/server-ca.crt: no such file or directory"
Aug 19 08:48:11 control-0 rancher-system-agent[3417]: time="2022-08-19T08:48:11-06:00" level=error msg="error while appending ca cert to pool for probe kube-apiserver"
```

This seems to be similar to #37802 but I am not changing the `data-dir` property. Everything is strictly created from the UI

**Comment by cbron** [comment]
AlmaLinux is not a supported OS. You can see our supported downstream RKE2 OS's here: https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-7/

**Comment by rajha-korithrien** [comment]
There is very little difference between AlmaLinux and Rocky.
I have repeated the experiment with Rocky as well, same behavior.

It turns out the issue is is with the "minimal install" part. A minimal install of Rocky/AlmaLinux is the fact that they do not install "tar" by default.

A brief forum discussion of this same issue for Rocky can be found here: https://forums.rockylinux.org/t/tar-not-pre-installed-on-rocky-minimal-version/5341/2

As soon as I did `dnf install tar` the node came up.

We don't observe this issue when using a minimal node and using the RKE2 "rpm" based installer because that install method uses yum repos and RPMS. The only reason `rancher-system-agent` method is failing is because it does a `tarball` based install on an RPM based system, and there is no tar command to be found on the host.
